tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_proactive_IPI.py

```diff
@@ -68,9 +68,14 @@ def test_automated_recovery_from_failed_nodes_IPI_proactive(
         # Check the pods should be in running state
         all_pod_obj = pod.get_all_pods(wait=True)
         for pod_obj in all_pod_obj:
-            wait_for_resource_state(
-                resource=pod_obj, state=constants.STATUS_RUNNING, timeout=200
-            )
+            # 'rook-ceph-crashcollector' on the failed node stucks at pending
```

Comment:
In the current WA, I am only skipping the health check of 'rook-ceph-crashcollector'.
Lets say if any of the next test cases does health_check for all pods in openshift-storage project with status 'running' then that test might fail since 'rook-ceph-crashcollector' stays at pending state for ever.

So, I think it would be better to delete the 'rook-ceph-crashcollector' associated deployment, so that the pending pod disappears.

Marking this a WIP to have it tested and implement.

Reply:
done